src/main/java/jarvis/logic/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package jarvis.logic;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+
+    LocalDate dateDeadline;
+    String deadline;
+    public Deadline(String name, String deadline){
```

Comment:
Perhaps leave a whitespace before '{' ? I noticed the same issue in a few other places too!